packages/shorebird_cli/lib/src/os/operating_system_interface.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:io';
+
+import 'package:mason_logger/mason_logger.dart';
+import 'package:scoped/scoped.dart';
+import 'package:shorebird_cli/src/platform.dart';
+import 'package:shorebird_cli/src/process.dart';
+
+// TODO(bryanoltman): remove this once os is used.
+// coverage:ignore-start
+/// A reference to a [OperatingSystemInterface] instance.
+final osRef = create(OperatingSystemInterface.new);
+
+/// The [OperatingSystemInterface] instance available in the current zone.
+OperatingSystemInterface get os => read(osRef);
```

Comment:
seems weird to call this `os` when the class isn't called `OperatingSystem` imo

Reply:
Fixed